index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt(`Ingrese su nombre`)
+alert(`Hola ${nombreUsuario}`)
```

Comment:
No creo que deberías utilizar interpolación de strings en este caso.
```suggestion
alert(`Hola` + nombreUsuario)
```